### PR TITLE
[EP-762] Saving settings to persistent storage 

### PR DESCRIPTION
**Pull request by AndreiGalkin**

### CODE REVIEW

File: sample-app/screens/Settings.tsx [hunk 1]
```diff
@@ -20,15 +21,42 @@ import {
...
 
+  /**
+  * Lifecycle
+  */
+  componentDidMount() {
+    this.getData()
+  }
+
+  componentWillUnmount() {
+  store.dispatch({ type: '' })
```
Comment (stpopa) [inline sample-app/screens/Settings.tsx]: seems like formatting is wrong in here

File: sample-app/screens/Settings.tsx [hunk 2]
```diff
@@ -20,15 +21,42 @@ import {
...
+  }
+
+  async storeData(data: any) {
+    try {
+      await AsyncStorage.setItem(storageKey, JSON.stringify(data)).then()
+    } catch (e) { }
```
Comment (stpopa) [inline sample-app/screens/Settings.tsx]: can we log the error?

File: sample-app/screens/Settings.tsx [hunk 3]
```diff
@@ -20,15 +21,42 @@ import {
...
+  }
+
+  async getData() {
+    try {
+      const value = await AsyncStorage.getItem(storageKey)
+      if(value !== null) {
+        this.setState({ settingsData: JSON.parse(value) })
+      }
+    } catch(e) { }
```
Comment (stpopa) [inline sample-app/screens/Settings.tsx]: can we log the error?

File: sample-app/screens/Settings.tsx (hunk 1 above)
Comment (AndreiGalkin) [reply sample-app/screens/Settings.tsx]: 👍

File: sample-app/screens/Settings.tsx (hunk 2 above)
Comment (AndreiGalkin) [reply sample-app/screens/Settings.tsx]: 👍

File: sample-app/screens/Settings.tsx (hunk 3 above)
Comment (AndreiGalkin) [reply sample-app/screens/Settings.tsx]: 👍

File: sample-app/screens/SettingsConfig.tsx [hunk 4]
```diff
@@ -1,4 +1,11 @@
 import React from 'react';
+import { createStore } from 'redux'
+
+function counter() { }
```
Comment (stpopa) [inline sample-app/screens/SettingsConfig.tsx]: seems like this is unused

**Pull request merged**